Shared/Services/WolService.cs

```diff
@@ -125,29 +125,21 @@ private static byte[] GetWolPacket(byte[] macAddress)
 
     private static void BroadcastWol(IPAddress localAddress, IPAddress broadCastAddress, byte[] data)
     {
-        //var localEP = new IPEndPoint(localAddress, 0);
-        var udpc = new UdpClient();
-
+        using var udpc = new UdpClient(new IPEndPoint(localAddress, 0));
```

Comment:
For some reason, not binding to an address causes sometimes the client not to send the Datagram.
Spamming a WOL Action does not throw an Exception, the Socket gets released early enough, but maybe for safety add ReuseAddr Option